src/plugins/core/cell.ts

```diff
@@ -556,8 +554,10 @@ export class CellPlugin extends CorePlugin<CoreState> implements CoreState {
       style,
       format,
       isFormula: true,
-      compiledFormula,
-      dependencies: [],
+      compiledFormula: {
+        ...compiledFormula,
+        dependencies: [],
+      },
```

Comment:
spread and `dependencies: []` not needed.  We know `compiledFormula.dependencies` is already an empty array at this point.

Reply:
Needed for typing, compiledFormula.dependencies is as `Array<string>`, and we want an `Array<Range>` in the ForumlaCell :x